utils.js

```diff
@@ -2,7 +2,10 @@
 
 const { Gio } = imports.gi;
 
-const Extension = imports.misc.extensionUtils.getCurrentExtension();
+const ExtensionUtils = imports.misc.extensionUtils;
+const Extension = ExtensionUtils.getCurrentExtension();
```

Comment:
nit: you don't use `Extension` here so we could drop this line but doesn't hurt.